front-end/src/hooks/authContext.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState,  useContext, createContext } from 'react';
+import PropTypes from 'prop-types';
+import Civic from '../services/Civic';
+
+const authContext = createContext();
+
+function useProvideAuth() {
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+    let civic = new Civic(); 
+
+    const accountLogin = async ({username, password}) => {
+        try{

```

Comment:
why not throw an error if it is throw, not return silently with return error value?